src/dll/edbgEcmdDll.C

```diff
@@ -1696,11 +1702,263 @@ uint32_t dllQueryHostMemInfoRanges( const std::vector<ecmdChipTarget> & i_target
 #endif // ECMD_REMOVE_MEMORY_FUNCTIONS
 
 #ifndef ECMD_REMOVE_INIT_FUNCTIONS
```

Comment:
I've got my head around the first part of this I would like to see changed.  The `#indef ECMD_REMOVE_INIT_FUNCTIONS` should only be around the ecmd dllIstep* functions, since that is what it is designed to control.  It shouldn't be around this other implementation code.   Still need to work out the new define.